misc/userscripts/qute-pass

```diff
@@ -134,19 +134,21 @@ def find_pass_candidates(domain, password_store_path):
     return candidates
 
 
-def _run_pass(pass_arguments, encoding):
+def _run_pass(pass_arguments, encoding, password_store):
     # The executable is conveniently named after it's mode [pass|gopass].
     pass_command = [arguments.mode]
-    process = subprocess.run(pass_command + pass_arguments, stdout=subprocess.PIPE)
+    env = os.environ.copy()
+    env['PASSWORD_STORE_DIR'] = password_store
```

Comment:
I haven't tested it  locally, but I think you can also access `arguments.password_store` directly, instead of passing it via `pass_()` and `pass_otp()`, no?

Reply:
Given that we already pass `encoding` as commandline argument as well, I think it makes more sense to be consistent here.

Reply:
@bascht: You're right though, arguments is defined outside the main scope, so it would be possible. We could alternatively replace both `encoding` and `password_store` with `arguments` attributes.

It would also be possible to use the parsed/expanded variable `password_store_path`. Shouldn't make any difference in this case, but it would be more consistent to use the exact same string for all use cases. This won't be possible if `arguments` is used, unless of course it is modified.

Reply:
Good point, I think indeed using `password_store_path` would be a good idea. Right now, if someone spawned `qute-pass` with `--password-store ~/some/path` from qutebrowser, things most likely still wouldn't work properly, because the `~` wouldn't get expanded in the environment variable.

Reply:
I made a commit now in which the password store path is expanded in the argument parsing process, regardless of its source, so that both `arguments.io_encoding` and `arguments.password_store` can be accessed directly when needed. Less duplication of concerns, and more consistent code.